domain/globalconfigsync/globalconfig.go

```diff
@@ -17,58 +17,37 @@ package globalconfigsync
 import (
 	"context"
 
-	"github.com/pingcap/tidb/ddl/util"
 	"github.com/pingcap/tidb/util/logutil"
-	"go.etcd.io/etcd/clientv3"
+	pd "github.com/tikv/pd/client"
 	"go.uber.org/zap"
 )
 
-const (
-	globalConfigPath     = "/global/config/"
-	keyOpDefaultRetryCnt = 5
-)
-
-// GlobalConfigSyncer stores global config into etcd.
+// GlobalConfigSyncer is used to sync pd global config.
 type GlobalConfigSyncer struct {
-	etcdCli  *clientv3.Client
-	NotifyCh chan ConfigEntry
+	pd       pd.Client
+	NotifyCh chan pd.GlobalConfigItem
 }
 
-// NewGlobalConfigSyncer returns a new GlobalConfigSyncer.
-func NewGlobalConfigSyncer(etcdCli *clientv3.Client) *GlobalConfigSyncer {
-	return &GlobalConfigSyncer{
-		etcdCli:  etcdCli,
-		NotifyCh: make(chan ConfigEntry, 8),
+// StoreGlobalConfig is used to store global config.
```

Comment:
Why change the function order? It makes the PR look like have a big change, but it doesn't.